tests/test_utils.py

```diff
@@ -486,3 +492,167 @@ def template(x):
             template(3)
 
         assert str(recorder) == expected_output
+
+
+@qml.template
+def dummy_template(wires):
+    """Dummy template for inv tests."""
+    for wire in wires:
+        qml.RX(1, wires=[wire])
+        qml.RY(-1, wires=[wire])
+
+
+def inverted_dummy_template_operations(wires):
+    """The expected inverted operations for the dummy template."""
+    return sum(
+        [
+            [qml.RY(-1, wires=[wire]).inv(), qml.RX(1, wires=[wire]).inv()]
+            for wire in reversed(wires)
+        ],
+        [],
+    )
```

Comment:
Would the following not work here (just a suggestion)?
```python
    for wire in reversed(wires):
        qml.RY(-1, wires=[wire]).inv()
        qml.RX(1, wires=[wire]).inv()
```

Reply:
Yes, that's much more readable!